backend/src/services/release.ts

```diff
@@ -295,6 +335,142 @@ export async function getReleaseBySemver(user: UserInterface, modelId: string, s
   return release
 }
 
+function getSemverQueryBounds(querySemver: string) {
+  const semverRangeStandardised = semver.validRange(querySemver, { includePrerelease: false })
+
+  if (!semverRangeStandardised) {
+    throw BadReq(`Semver range, '${querySemver}' is invalid.`)
```

Comment:
In general with errors, we try and avoid injecting variables into the messages. Instead you can pass in a context object containing any relevant data. Eg `throw BadReq(`Semver range is invalid.`, {semverQuery: querySemver})`

Reply:
Changed